### PR TITLE
Do not add SBD playbook for native fencing

### PR DESCRIPTION
**Pull request by lpalovsky**
This PR removes SBD playbook from default HANA setup list and adds it only in 
case of HA scenario with "FENCING_MECHANISM" set to "sbd".

- Related ticket: https://jira.suse.com/browse/TEAM-7174
- Needles: NA
- Verification run: 
Native fencing: https://openqaworker15.qa.suse.cz/tests/124284/#
SBD fencing: http://openqaworker15.qa.suse.cz/tests/124037#

VR test does not need to pass here, important is correct ansible playbook list being compiled. Check yaml files in logs.

### CODE REVIEW

File: tests/sles4sap/publiccloud/qesap_terraform.pm [hunk 1]
```diff
@@ -58,29 +58,35 @@ sub set_var_output {
...
         my $reg_playbook = "registration.yaml -e reg_code=" . get_required_var("SCC_REGCODE_SLES4SAP") . " -e email_address=''";
         push(@playbook_list, $reg_playbook);
     }
 
-    if ($ha_enabled == 1) {
+    # SLES4SAP/HA related playbooks
+    if ($ha_enabled) {
+        push(@hana_playbook_list, "pre-cluster.yaml");
+        push(@hana_playbook_list, "sap-hana-preconfigure.yaml -e use_sapconf=" . set_var_output("USE_SAPCONF", "true"));
+        if (get_var("FENCING_MECHANISM") eq "sbd") { push(@hana_playbook_list, "cluster_sbd_prep.yaml"); }
```
Comment (mpagot) [inline tests/sles4sap/publiccloud/qesap_terraform.pm:73]: push(@hana_playbook_list, "cluster_sbd_prep.yaml")     if (get_var("FENCING_MECHANISM") eq "sbd") ;

**Review by mpagot — COMMENTED** [review]
LGTM

File: tests/sles4sap/publiccloud/qesap_terraform.pm (hunk 1 above)
Comment (alvarocarvajald) [reply tests/sles4sap/publiccloud/qesap_terraform.pm:73]: +1 to Michele's suggestion, but also using `check_var()`:
```
push(@hana_playbook_list, "cluster_sbd_prep.yaml") if (check_var('FENCING_MECHANISM', 'sbd') );
```

**Review by alvarocarvajald — APPROVED** [review]
Left a comment, but in general LGTM

File: tests/sles4sap/publiccloud/qesap_terraform.pm (hunk 1 above)
Comment (lpalovsky) [reply tests/sles4sap/publiccloud/qesap_terraform.pm]: Oh, such an ellegant line. I want to put it into gallery. 
Thanks both :)
Code updated.

**Pull request merged**